README.md

```diff
@@ -0,0 +1,222 @@
+
+- "Number of winners" - Per [Bylaws Section 5.5][2], 3 or 4 depending on
+  the outgoing cohort size for a given year. It *may* be more if any
+  additional board vacancies have been created.
+
+- "Shuffle candidate order": "yes"
+
+- There *may* be additional questions such as bylaws changes, if so
+  use the `(+) Add Another Contest` button and use judgement to determine
+  how to add them.
+
+### Setting up Voters
+
+The Election Adminitsrator (EA) will receive a file with the first name, last
+name, membership type, and email address of a voter.
+In order to add voters to an election,
+OpaVote requires they be uploaded in an ASCII text file in the format::
+
+    email-address
+
+For example::
+
+    graffatcolmingov@gmail.com
+    ewdurbin@pyfound.org
+
+The format of the file the election administrator typically gets is of the
+format::
+
+    FirstName,LastName,MembershipType,Email
+
+There is a tool in this repository to handle it called
+``convert-exported-csv-for-opavote.py``. This will generate the necessary
+format for OpaVote from the CSV file the EA receives.
+
+It's important to note here that not all voters have a first or last name and
+some don't have either.
+
+
+## Running an Election
+
+Running an election begins once the election starts. While an election is in
+progress, a few things may happen:
+
+- New voters may need to be added as they were missed when the original voter
+  list was generated
+
+- Some voters may need you to send/re-send their ballot notification email
+
+The following sub-sections will cover these in detail.
+
+### Adding Missed Voters
+
+OpaVote allows us to add new voters after an election has begun although
+the other details of the election or questions cannot be altered. The process for
+adding new voters is the same as is detailed above in "Setting up Voters". The
+EA will receive a new file full of the details of the voters who were missed.
+Using that, the EA can generate a new CSV to upload to OpaVote using the same
+tooling.
+
+### Re-sending Ballot Emails
+
+OpaVote provides statistics on pending, in-transit, rejected, and delivered
+ballot emails. It also tracks opens, clicks, and visits.
+
+If a reminder is necessary, it can be sent from the admin console by finding
+the voter and clicking "Send Ballot".
+
+If "Automatic reminders" was not selected during election setup,
+OpaVote will automatically send reminders every three days throughout
+the election period.
+
+## Closing an Election
+
+The end of an election requires the EA to verify the results of the election
+with Helios and then present the results to the Executive Director of the PSF.
+The Executive Director will then announce the results to the community.
+At that time, the `🌎 Publish Results` button on OpaVote can be used to
+publish the full results to all voters.
+
+### Achieving Quorum
+
+The PSF Bylaws_ state that one-third (1/3) of the members elligible to vote
+will constitute a quorum. See also:
+
+.. epigraph::
```

Comment:
I think this needs to not be reStructuredText

Reply:
https://github.com/python/psf-election/pull/6/commits/ac30982030fb061cc6925e3e2315db4f7ea04004